framework/resources/Functional/misc/describe1.e_tsv

```diff
@@ -10,7 +10,7 @@ dfs_describe	\{
       \"escape\" : \"`\",
       \"comment\" : \"\^\",
       \"skipFirstLine\" : true,
-      \"delimiter\" : \"|\"
+      \"delimiter\" : \"\|\"
```

Comment:
Not clear why we need this, either.

Reply:
'|' is an alternation symbol in regex, so it need to be escaped.